#!/usr/bin/env python
PACKAGE = "twos"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add("num1", double_t, 0, "A double parameter for first number", .1, 0,1000)
gen.add("num2", double_t, 0, "A double parameter for second number", .1, 0,1000)
gen.add("lines", str_t, 0, "A string parameter","twoss_dynamic_reconfigure")


exit(gen.generate(PACKAGE, "twos", "two_"))